### PR TITLE
Fixes # 22011: update javascript libraries

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/22011

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala [hunk 1]
```diff
@@ -216,8 +216,13 @@ object DisplayNode extends Loggable {
       JsRaw("$('#" + detailsId + "').tabs()") & {
         eltIds.map { i =>
           JsRaw(s"""
+<<<<<<< HEAD
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala:219]: merge problem

**Comment by ElaadF** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

**Review by VinceMacBuche — APPROVED** [review]

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Pull request merged**